build-tools/scripts/PrepareWindows.targets

```diff
@@ -9,7 +9,11 @@
   </PropertyGroup>
   <Import Project="$(_TopDir)\Configuration.props" />
   <Target Name="_BuildXAPrepare">
-    <MSBuild Projects="$(MSBuildThisFileDirectory)..\xaprepare\xaprepare.sln" Targets="Restore;Build" />
+    <MSBuild
+        Projects="$(MSBuildThisFileDirectory)..\xaprepare\xaprepare.sln"
+        Targets="Restore;Build"
+        RunEachTargetSeparately="True"
```

Comment:
What is the intention of using `RunEachTargetSeparately`? Why do we want to ignore failures in `Restore` target?

Reply:
It doesn't do what I thought it does, I'll be removing it when I bring it out of draft.